packages/react/src/utils/scriptLoader.ts

```diff
@@ -83,8 +78,12 @@ export async function loadScript(params: LoadScriptParams): Promise<HTMLScriptEl
     }
 
     const script = document.createElement('script');
-    script.setAttribute('data-clerk-publishable-key', publishableKey!);
-    script.setAttribute('data-clerk-frontend-api', frontendApi!);
+    if (publishableKey) {
+      script.setAttribute('data-clerk-publishable-key', publishableKey);
+    }
+    if (frontendApi) {
+      script.setAttribute('data-clerk-frontend-api', frontendApi);
+    }
```

Comment:
🔧 Let's throw on error if non of them is set at this point via the errorThrower.

Reply:
Thought of that as well, but it felt that this is too deep for an error as we already check a couple of times. Im going to add one as per your suggestion, better be on the safe side

Reply:
Done